llvm/test/CodeGen/AMDGPU/GlobalISel/legalize-build-vector.mir

```diff
@@ -299,129 +299,6 @@ body: |
     S_NOP 0, implicit %12
 ...
 ---
-name: legal_v13s32
```

Comment:
Why are these removed?

Reply:
Because these are not legal anymore

Reply:
But this is a legalizer test. The legalizer is supposed to legalize them.